content/docs/esc/faq.md

```diff
@@ -13,36 +13,34 @@ aliases:
 
-## Why did Pulumi launch ESC?
+## What is the pricing of Pulumi ESC?
+
+See our [pricing page](https://www.pulumi.com/pricing/) for details.
 
-We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets causing sprawl and duplications across stacks. Pulumi ESC is a natural extension of Pulumi IaC by allowing [hierarchical](/docs/esc/#configuration-as-code) configurations that can be used across stacks.
+## What counts as a secret towards pricing?
 
-## When will Pulumi ESC be generally available?
+Secret includes both [static secrets](/docs/esc/get-started/store-and-retrieve-secrets/), [dynamic login credentials](/docs/esc/integrations/dynamic-login-credentials/) and [dynamic secrets](/docs/esc/integrations/dynamic-secrets/).
 
-Pulumi ESC will be generally available in 2024.
+In other words,
 
-## What will be the pricing of Pulumi ESC?
+- When using the Pulumi ESC Document Editor, each definition of fn::secret:* and fn::open::* (except Pulumi-stacks provider) is counted as a secret.
+- When using the Pulumi ESC Table Editor, each secret and provider config type (except Pulumi-stacks) is counted as a secret.
 
-During the preview, the product is free to use. When the product is made generally available, it will have a pricing model that is competitive with the market.
+The number of secrets only from the latest environment revision is counted towards your billing.
 
-**What is the roadmap for Pulumi ESC?**
-<br>
-We have a lot of [features](https://github.com/pulumi/esc/issues) planned for ESC. Here are a few highlights:
+## What is the roadmap for Pulumi ESC?
 
-- [Webhooks](https://github.com/pulumi/esc/issues/188)
-- [Syncing config/secrets to external systems](https://github.com/pulumi/esc/issues/58)
-- [Version controlled environments](https://github.com/pulumi/esc/issues/63)
-- [Table UI authoring experience](https://github.com/pulumi/esc/issues/62)
+Check out our [public roadmap](https://github.com/orgs/pulumi/projects/44/) page.
 
-We encourage you to create a [feature request](https://github.com/pulumi/esc/issues/new/choose) or upvote [existing open requests](https://github.com/pulumi/esc/issues).
+If you are interested in a particular feature, we encourage you to create a [feature request](https://github.com/pulumi/esc/issues/new/choose) or upvote [existing open requests](https://github.com/pulumi/esc/issues).
 
-## Can a Self-hosted Pulumi customer use this?
+## Can I Self-host Pulumi ESC?
 
-Pulumi ESC is currently not supported for self-hosted customers. [Contact sales](/contact/?form=sales) if you'd like to participate in our self-hosted beta for ESC when it is available.
+Yes. [Contact sales](/contact/?form=sales) us to get a demo or trial self-hosted Pulumi ESC.
```

Comment:
is the `us` in this line supposed to be here?

Reply:
I'll update